ingenialink/canopen/network.py

```diff
@@ -1072,3 +1077,20 @@ def _get_servo_state(self, node_id: int) -> NET_STATE:
 
     def _set_servo_state(self, node_id: int, state: NET_STATE) -> None:
         self.__servos_state[node_id] = state
+
+    @staticmethod
+    def get_available_devices() -> List[str]:
+        if platform.system() == "Linux":
+            return [CAN_DEVICE.SOCKETCAN.name]
+        return list(
+            {
+                available_device["interface"]
+                for available_device in can.detect_available_configs(
+                    [
+                        device.value
+                        for device in CAN_DEVICE
+                        if device not in [CAN_DEVICE.SOCKETCAN, CAN_DEVICE.VIRTUAL]
```

Comment:
It's necessary to filter SOCKETCAN? you can simply ignore it and it will never be available in Windows.

Reply:
I think we should filter it, it makes no sense in Windows.

Reply:
I know, but can.detect_available_configs will filter it, right?
In addition, why function works different for windows and linux?

Reply:
Besides, an ugly exception occurs when we include it:
```
Traceback (most recent call last):
  File "C:\GitProjects\ingenialink-python\venv\lib\site-packages\can\interfaces\socketcan\utils.py", line 52, in find_available_interfaces
    output_str = subprocess.check_output(command, text=True)
  File "C:\Users\martin.acosta\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "C:\Users\martin.acosta\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Users\martin.acosta\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\martin.acosta\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```